### PR TITLE
Add explicit "translating" bit to translations

### PR DESCRIPTION
**Pull request by raisfathin**
This would allow leaders to explicitly state that they are not translating some tasks. Also allow staff to do the same on leaders' behalf as they are able to freeze leaders' translations already

Screenshots:
1. [User's home page (no frozen contest)](https://user-images.githubusercontent.com/9299240/147847596-07458f1a-4206-473c-9a75-d23722aafcd1.png)
2. [User's home page (1 frozen contest)](https://user-images.githubusercontent.com/9299240/147847598-c81e8f89-8e30-45c1-85b8-bdc0f702d2a9.png)
3. [Staff's users page](https://user-images.githubusercontent.com/9299240/147847636-1a349f5c-828b-47e5-bb09-b420579fffc4.png)
4. [Staff's user page](https://user-images.githubusercontent.com/9299240/147847601-d9632d7b-2c20-458e-91a9-094dcf6bf320.png)

~~Depends on #25~~

### CODE REVIEW

**Comment by fushar** [comment]
Can we rebase this?

**Comment by raisfathin** [comment]
Rebased.

File: trans/static/js/common.js [hunk 1]
```diff
@@ -53,34 +53,20 @@ function sendPrintJob(print_task_url) {
...
-	
-	if(check.checked) {
-		return confirm('Are you sure?');
-	}
-	
-    if(document.getElementsByName('reopen').length!= n) {
-        alert('All tasks must be finalized before this action!');
-        return false;
+function validateFinalizeTranslation(contest_slug, tasks_length) {
+    const frozen_tasks = document.querySelectorAll(
```
Comment (fushar) [inline trans/static/js/common.js:57]: Nit: indentation looks off? (Mix between tabs and spaces in this file)

File: trans/templates/home.html [hunk 2]
```diff
@@ -37,7 +37,11 @@ <h2>{{ contest.title }}</h2>
                 </thead>                
                 <tbody>
                 {% for task in contest.tasks %}
-                    <tr>
+                  <tr
+		    class="task-row"
```
Comment (fushar) [inline trans/templates/home.html:41]: Nit: indentation looks off? (Mix between tabs and spaces in this file)

**Review by fushar — CHANGES_REQUESTED** [review]
Any screenshot? 😄

File: trans/static/js/common.js (hunk 1 above)
Comment (raisfathin) [reply trans/static/js/common.js:57]: Done.

File: trans/templates/home.html (hunk 2 above)
Comment (raisfathin) [reply trans/templates/home.html]: Done.

**Comment by raisfathin** [comment]
Added screenshots to comment #1.

File: trans/static/js/common.js [hunk 3]
```diff
@@ -43,8 +43,8 @@ function sendPrintJob(print_task_url) {
         type: "POST",
         success: function (response) {
             ToastrUtil.success(
-                    'Printouts will be put to the DRAFTs desk for YOUR PICK-UP shortly.',
-                    'Print job submitted.'
+                'Printouts will be put to the DRAFTs desk for YOUR PICK-UP shortly.',
```
Comment (fushar) [inline trans/static/js/common.js:46]: Super duper nit: revert indentation

File: trans/templates/home.html [hunk 4]
```diff
@@ -6,112 +6,168 @@
 
 <script type="text/javascript">
     if(performance.navigation.type == 2) {
-		location.reload(true);
-	}
+                location.reload(true);
```
Comment (fushar) [inline trans/templates/home.html:9]: Same

**Review by fushar — APPROVED** [review]

File: trans/static/js/common.js (hunk 3 above)
Comment (raisfathin) [reply trans/static/js/common.js]: Done.

File: trans/templates/home.html (hunk 4 above)
Comment (raisfathin) [reply trans/templates/home.html]: Done.

**Pull request merged**